stubs/CoreGenericFunctions.phpstub

```diff
@@ -1086,6 +1086,31 @@ function chunk_split(string $str, int $chunklen = 76, string $end= ''): string
 {
 }
 
+/**
+ * @template TInput as \Countable|\SimpleXMLElement|\ResourceBundle|array
+ *
+ * @param TInput $value
+ * @param 0|1 $mode
+ *
+ * @return (
+ *          TInput is array<empty, empty>
+ *          ? 0
+ *          : (TInput is array{0: mixed, 1: mixed}
```

Comment:
This match is wrong, since it will also match `array{0: mixed, 1: mixed, 2: mixed}`, which would then result in `2` being inferred.